step-capstone/src/components/TripForms/AddTrip.js

```diff
@@ -24,28 +24,33 @@ const useStyles = makeStyles({
 })
 
 
-export default function AddTrip (props) {
+export default function AddTrip(props) {
     const classes = useStyles();
     const newTrip = {
         title: "",
         startDate: new Date(),
         endDate: new Date(),
-        destinations: "",
+        destination: null,
         description: "",
+        userPref: {
+            budget: 2,
+            radius: 10,
```

Comment:
default radius is 10 meters?

Reply:
It's in kilometers for the sake of the slider haha. I put the slider in kilometers because a max of 50000 meters is a lot of increments, i just convert the km into m when getting suggestions

Reply:
great!